README.adoc

```diff
@@ -63,45 +65,45 @@ You should find the *uber jar* of the collector inside your `target` folder.
+Internally the collector maintains a cache of the metrics it receives. The default for metric emission is every 60 seconds SCSt 2.x and 5 seconds for SCSt 1.x applications, but can be tuned on the application by using Spring Boot's metrics exporter scheduling control, please refer to the docs https://docs.spring.io/spring-cloud-stream/docs/Elmhurst.RELEASE/reference/htmlsingle/index.html#spring-cloud-stream-overview-metrics-emitter[here] to configure your applications.
+
+The default for the collector is to evict any metric reading that was not updated over the past 90 seconds. You can however change this value by setting the property `spring.cloud.dataflow.metrics.collector.evictionTimeout`
 
-The default for the collector is to evict any metric reading that was not updated over the past 30 seconds.
-You can however change this value by setting the property `spring.cloud.dataflow.metrics.collector.evictionTimeout`
+Note: It is important that the eviction time is set to a value higher thant the emission time!
 
 == Security
 
-The collector will have security enabled by default, make sure you start it with a proper username/password using `--security.user.name` and `--security.user.password`
+The collector will have security enabled by default, make sure you start it with a proper username/password using `--spring.security.user.name` and `--spring.security.user.password`
 
 == E2E Cheat sheet
 
 The following is just a sample of commands that one can use to get the collector up and running and see some metrics on the dataflow UI.
 
 ```
 Collector:
-java -jar target/metrics-collector-rabbit-1.0.0.BUILD-SNAPSHOT.jar --security.user.name=spring --security.user.password=cloud
+java -jar target/metrics-collector-rabbit-2.0.0.BUILD-SNAPSHOT.jar --spring.security.user.name=spring --spring.security.user.password=cloud
 
 Server:
-java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-1.2.0.BUILD-SNAPSHOT.jar --spring.cloud.dataflow.metrics.collector.uri=http://localhost:8080 --spring.cloud.dataflow.metrics.collector.username=spring --spring.cloud.dataflow.metrics.collector.password=cloud
+java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-1.5.0.BUILD-SNAPSHOT.jar --spring.cloud.dataflow.metrics.collector.uri=http://localhost:8080 --spring.cloud.dataflow.metrics.collector.username=spring --spring.cloud.dataflow.metrics.collector.password=cloud
 
-Register Apps:
-app import --uri http://bit.ly/Bacon-RELEASE-stream-applications-rabbit-maven
+Register SCSt 1.x Apps:
+app import --uri http://bit.ly/Celsius-SR1-stream-applications-rabbit-maven
 
+Or Register SCSt 2.x Apps: TODO
```

Comment:
We can remove it for now. Once the App release is done, we can add it with links.